package.json

```diff
@@ -26,6 +26,8 @@
   },
   "dependencies": {
     "chalk": "^1.1.3",
-    "minimist": "^1.2.0"
+    "htmltojsx": "git://github.com/harrison/react-magic.git",
```

Comment:
Great! I have a few other improvements in mind. I'm using it in a soon-to-be-live app so happy to contribute as much as possible. 

Are you happy with the "raise issue" -> "submit PR" flow? I'm not sure if there is a "enhancement" tag in the issues area. Or would you prefer straight to PR? Raising an issue first probably allows you to pre-vet possible enhancements incase I am too off base with a PR. Thoughts?